lib/applicationProcessing/resolvers.ts

```diff
@@ -0,0 +1,237 @@
+import { ApolloError } from 'apollo-server-errors'; // Apollo error
+import { Resolver } from '@lib/resolvers'; // Resolver type
+
+/**
+ * Updates the ApplicationProccessing object with the optional values provided
+ * @returns Status of operation (ok, error)
+ */
+export const updateApplicationProcessing: Resolver = async (_, args, { prisma }) => {
+  const { input } = args;
+  const { id, documentUrl, ...rest } = input;
+
+  let applicationProccessing;
+  try {
+    applicationProccessing = await prisma.applicationProcessing.update({
+      where: { id: parseInt(id) },
+      data: {
+        ...rest,
+        ...(documentUrl && {
+          documentUrls: {
+            push: documentUrl,
+          },
```

Comment:
Can you add TODO comment for figuring out deleting files? Will be addressed Post-MVP